unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/ExploreHUD/Scripts/HighlightScenes/HotSceneCellView.cs

```diff
@@ -117,16 +118,22 @@ public void Clear()
     public void JumpInPressed()
     {
         HotScenesController.HotSceneInfo.Realm realm = new HotScenesController.HotSceneInfo.Realm() { layer = null, serverName = null };
+        hotSceneInfo.realms = hotSceneInfo.realms.ToList().OrderByDescending(x => x.usersCount).ToArray();
```

Comment:
Until now, the Explore's JumpIn was working exactly the opposite of how it should! The `hotSceneInfo.realms` list was ordered so that the realms with the fewest users were at the top of the list, then we always were jumping to the worst option. Now I have sorted the list by number of users in a descending order and it already works as it should 👍